Jellyfin.Api/Helpers/TranscodingJobHelper.cs

```diff
@@ -530,6 +531,15 @@ private void DeleteHlsPartialStreamFiles(string outputFilePath)
             {
                 var attachmentPath = Path.Combine(_appPaths.CachePath, "attachments", state.MediaSource.Id);
                 await _attachmentExtractor.ExtractAllAttachments(state.MediaPath, state.MediaSource, attachmentPath, CancellationToken.None).ConfigureAwait(false);
```

Comment:
Is it necessary to extract attachments when the file is external?

Reply:
People make mistakes and so it can happen that fonts are attached to the wrong file. I think that we should get whatever font we can. mpv behaves in the same way and people usually test their videos with mpv.